File: Csharp/Lario and Muigi Pipe Problem.cs
// https://www.codewars.com/kata/56b29582461215098d00000f/train/csharp
using System.Collections.Generic;
using System.Linq;

public class Fixer
{
  public static List<int> PipeFix(List<int> numbers)
  {
    return Enumerable.Range(numbers.First(), numbers.Last() - numbers.First() + 1).ToList();
  }
}

File: javascript/Lario and Muigi Pipe Problem.js
// https://www.codewars.com/kata/56b29582461215098d00000f/train/javascript

function pipeFix(numbers){
    let res = []
    const first = numbers[0]
    const last = numbers[numbers.length-1]
    for(let i = first; i <= last; i++){
      res.push(i)
    }
      return res 
    }